src/utils/ui.js

```diff
@@ -14,7 +15,7 @@ export function isVisible() {
 
 /* Gets the line numbers width in the code editor
  */
-export function getLineNumberWidth(editor) {
+export function getLineNumberWidth(editor: Object) {
```

Comment:
Let's stick to `BreakpointsDropdown.js` file in this PR, we can have another PR for this one :)